addons/cloud_storage_google/__init__.py

```diff
@@ -0,0 +1,28 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import models
+from odoo.exceptions import UserError
+from odoo.tools import SQL
+from .models.cloud_storage_google import GOOGLE_CLOUD_STORAGE_ENDPOINT
+
+
+def uninstall_hook(env):
+    cr = env.cr
+    cr.execute(SQL(
+        """
+            SELECT type
+            FROM ir_attachment
+            WHERE type = 'cloud_storage'
+            AND url LIKE %s
+            LIMIT 1
+        """,
+        GOOGLE_CLOUD_STORAGE_ENDPOINT + '/%%'
```

Comment:
The string on the right-hand side of `LIKE` does not need to be escaped
```suggestion
        GOOGLE_CLOUD_STORAGE_ENDPOINT + '/%'
```
The escaping is only necessary when you inline it inside the SQL code.